microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -0,0 +1,11 @@
+[id="microshift-4-15-release-notes"]
```

Comment:
I was back and forth on whether this stub file needs the new [`mod-docs-content-type`](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#:~:text=//%20Module%20included%20in%20the%20following%20assemblies%3A%0A//%0A//%20*%20list%20of%20assemblies%20where%20this%20module%20is%20included%20%20%20%20%20%20%20%20%20%20%20%20%20%20(1)%0A%0A%3A_mod%2Ddocs%2Dcontent%2Dtype%3A%20%3CTYPE%3E%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20(2)%0A%5Bid%3D%22%3Cmodule%2Danchor%3E_%7Bcontext%7D%22%5D%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20(3)%0A%3D%20Module%20title)

Reply:
CC @kalexand-rh I'm _think_ the answer is no because this stub file isn't a module or an assembly, but wanted to check.

Reply:
In the end, it probably will need the content type. It's technically an assembly.

Reply:
Except none of these have a content type field at all, they just begin with an ID and a heading.
[id="ocp-4-14-release-notes"]
= {product-title} {product-version} release notes
include::_attributes/common-attributes.adoc[]
:context: release-notes

I'll put add it to this one and we can test, @kalexand-rh?